docs/release_guide.md

```diff
@@ -2,10 +2,10 @@ In order to release a new version of `demisto-sdk` to the public follow these st
 
 ### Validation before release:
 1) Make sure the **CHANGELOG.md** file is in order and is updated with all the changes in the current release.
-2) Create a new release branch on the sdk repo, formatted as `X.X.X`, e.g. `1.0.0`.
+2) Create a new release branch on the sdk repo, formatted as `X.X.X`, e.g. `1.0.0` (push the branch to the remote). 
 3) Make sure that both `sdk-nightly` and `sdk-master` builds passed.
-   * If **new SDK commits** were pushed after the nightly tests had started, manually trigger the sdk nightly build again as written in step 4. This will test sdk master on content branch.
-   * If **no new SDK commits** were done after the nightly tests, skip step 4.
+   * If **new SDK commits** were pushed after the nightly tests had started, manually trigger the sdk nightly build again as written in steps 4 + 5. This will test thr sdk release branch on content branch.
```

Comment:
```suggestion
   * If **new SDK commits** were pushed after the nightly tests had started, manually trigger the sdk nightly build again as written in steps 4 + 5. This will test the sdk release branch with content master.
```

Reply:
it's a new content branch - in step 4 we should open a new branch and update the SDK version in the requirements file to the release version. The branch's content is identical to master... but it is still a different branch so I'm not sure what would be less confusing to write here.